packages/synthetics/manifest.yml

```
@@ -30,7 +30,7 @@ conditions:
       - "uptime"
     subscription: "basic"
   kibana:
-    version: "^8.11.0"
+    version: "^8.17.0"
```

```suggestion
    version: "^8.11.0"
```

we should fix this as back as possible